packages/next-sample/libraries/logto.ts

```diff
@@ -0,0 +1,13 @@
+import NodeClient from '@logto/next';
```

Comment:
```suggestion
import LogtoClient from '@logto/next';
```

Reply:
Here may be confusing, for we have a `LogtoClient` in next SDK, too.